app/loaders/gtfsDepartures.tsx

```diff
@@ -1,6 +1,91 @@
-export default async function displayStopIds(params: any) {
+export default async function gtfsDepartures({params}) {
+  const {stopIds} = params
+    /* query for info stop ID*/
+    // const query = `{
+    //     allGtfsStops(filter: {id: {in: ["${stopIds}"]}}) {
+    //       nodes {
+    //         name
+    //         id
+    //         gtfsConnectionsByStopId {
+    //           nodes {
+    //             gtfsRouteByRouteId {
+    //               color
+    //               longName
+    //               shortName
+    //               textColor
+    //             }
+    //           }
+    //         }
+    //       }
+    //     }
+    //   }
+    //   `;
+
+
+    const query = `{
```

Comment:
remove this query and just keep query2 but rename it as query 
The first one was just an introduction, you won't need it